app/templates/admin/manageDepartments.html

```diff
@@ -93,4 +94,43 @@ <h1>Manage Departments</h1>
         <div class="col-md-2"></div>
     </div>
 </div>
+<div class="modal" id="manageDepartmentSupervisorModal" tabindex="-1" role="dialogue" >
+  <div class="modal-dialog" role="document">
+    
+    
+    <div id="manageSupervisorContent" class="modal-content">
+      
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <div class="department-name"></div>
+        
+      </div>
+
+      <div class="modal-body ">
+        <div class="form-group">
+        <label for="supervisorModalSelect">Select a supervisor to add:</label>
+        <div class="dropdown"><select class="selectpicker"
+          name='supervisorId'
+          id='supervisorModalSelect'
+          data-live-search='true'
+          title="Supervisor">
```

Comment:
I'd say that for a single tag like the select that you have here we should be consistent with whether we're using single or double quotes for the attributes. I think it's fine if different tags have different styles but the inconsistency within the tag is iffy to me. Up to you